staging/src/k8s.io/apimachinery/pkg/util/wait/wait.go

```diff
@@ -380,7 +401,9 @@ func WaitFor(wait WaitFunc, fn ConditionFunc, done <-chan struct{}) error {
 // timeout has elapsed and then closes the channel.
 //
 // Over very short intervals you may receive no ticks before the channel is
-// closed. A timeout of 0 is interpreted as an infinity.
+// closed. A timeout of 0 is interpreted as an infinity, and in such a case
+// it would be the caller's responsibility to close the done channel.
```

Comment:
this comment doesn't make sense to me... there is no done channel passed in or returned